wormhole/module.py

```diff
@@ -125,6 +139,16 @@ async def set_wormhole_channel(
             )
             return
 
+        delay = storage.get(self, 0, key="wormhole_slowmode")
+        try:
+            await channel.edit(slowmode_delay=delay)
+        except discord.Forbidden:
+            await guild_log.warning(
+                None,
+                itx.channel,
+                "I do not have proper permissions to set ongoing slow mode.",
```

Comment:
The bot does not talk 😀

The message should look like "Can't set slow mode for wormhole channel `channel`."